### ISSUE TITLE
Tab alternation not quite right...

### ISSUE THREAD

**Issue by arr2036** [alandekok/recli#8]
```text
show (session (apn STRING [([active]|[count])]|(apn STRING [count]|mdn INTEGER) [active])|subscriber id INTEGER)
```

```text
show session
...>
<tab>
...> apn
<tab>
...> apn
<tab>
...> mdn
<tab>
...>
```

You can see there's two paths that involve an apn node, which is probably why it's showing an apn completion twice.  The stupid simple hack would be to just strcmp current and next completions and skip until one is found that doesn't match.

**Comment by alandekok** [comment]
> You can see there's two paths that involve an apn node, which is probably why it's showing an apn completion twice

Yes.  The solution is a similar one to #5.

The command should have the `apn` string in front of the alternation:

    show (session (apn (STRING [([active]|[count])]|(STRING [count] ...)))

This means checking the alternation recursively.  I've got some code started on that.

**Comment by alandekok** [comment]
This is fixed in commit ca9ca469

**Comment by arr2036** [comment]
Confirmed fixed
